install_config/persistent_storage/topics/glusterfs_overview_standalone.adoc

```diff
@@ -0,0 +1,17 @@
+If you have a standalone {gluster} cluster available in your environment you
+can make use of volumes on that cluster using {product-title}'s GlusterFS
+volume plug-in. This solution is a conventional deployment where applications
+run on dedicated compute nodes, an {product-title} cluster, and storage is
+provided from it sown dedicated nodes.
+
+.Architecture - Standalone {gluster} Cluster Using {product-title}'s GlusterFS Volume Plug-in
+image::OpenShift_Containerization_Gluster_412816_0716_JCS_dedicated.png["Architecture - Standalone {gluster} Cluster Using {product-title}'s GlusterFS Volume Plug-in"]
+
+See the link:{gluster-install-link}[{gluster} Installation Guide] and the
+link:{gluster-admin-link}[{gluster} Administration Guide] for more on {gluster}.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
```

Comment:
In this instance, it should be "High availability" since this refers to the concept of making the service continuously available.